src/modules/tableValues.js

```diff
@@ -29,11 +40,31 @@ const getFormattedCellValue = ({ value, header = {}, visualization = {} }) => {
     }
 
     if ([VALUE_TYPE_DATE, VALUE_TYPE_DATETIME].includes(header.valueType)) {
+        let valueType = header.valueType
```

Comment:
On line 72 I see `header?.valueType` - we should figure out if `?` is needed everywhere or can be removed

Reply:
I think it can be removed since `header` is initialised to empty object anyway on line 29.

Reply:
Removed.